.github/workflows/release.yml

```
@@ -125,11 +125,68 @@ jobs:
         with:
           name: ipa-files
           path: build/ios/iphoneos/*.ipa
+  
+  build_windows:
+    name: Build Release Windows Installer
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)